src/store/sidebarStore.js

```diff
@@ -36,23 +30,6 @@ const mutations = {
 	hideSidebar(state) {
 		state.show = false
 	},
-	/**
-	 * Renaming state of the conversation
-	 *
-	 * @param {object} state current store state;
-	 * @param {boolean} boolean the state of the renaming action;
-	 */
-	isRenamingConversation(state, boolean) {
-		if (boolean) {
-			// Record sidebar status before starting editing process
-			state.sidebarOpenBeforeEditing = state.show
-			state.isRenamingConversation = true
-		} else {
-			state.isRenamingConversation = false
-			// Go back to the previous sidebar state
-			state.show = state.sidebarOpenBeforeEditing
-		}
-	},
```

Comment:
We've never been here, I suppose :) 

![image](https://github.com/nextcloud/spreed/assets/84044328/8fcacc69-1425-47f7-beaa-dac232688ece)

Reply:
> We've never been here, I suppose :)

Leftovers from features 3-4 years ago